cluster-up/up.sh

```diff
@@ -9,25 +9,10 @@ fi
 
 source ${KUBEVIRTCI_PATH}hack/common.sh
 source ${KUBEVIRTCI_CLUSTER_PATH}/$KUBEVIRT_PROVIDER/provider.sh
-
-KUBEVIRT_STORAGE=rook-ceph-default
-
 up
 
 # check if the environment has a corrupted host
 if [[ $(${KUBEVIRTCI_PATH}kubectl.sh get nodes | grep localhost) != "" ]]; then
     echo "The environment has a corrupted host"
     exit 1
 fi
-
```

Comment:
what is the purpose of this file now? does it do anything?

Reply:
This actually starts the cluster (`up`) and it is part of `kubevirtci`. You can find exactly the same file in kubevirt or cdi. I want to keep the code that is provided by kubevirtci clean without any changes, and do any additional logic and hacks in `hack/` directory. That is why I moved some code from here to `hack/cluster-up.sh`. 

I am not sure I understand everything about kubevirtci, but I think that now I can easily update kubevirtci to newer version without worrying about merge conflicts.

Reply:
So I want keep the whole `cluster-up` directory clean without kubevirt-velero-plugin changes.